### ISSUE TITLE
Fix false negatives when `fix` is missing and `computedEditInfo` is set to `true`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/jest-preset-stylelint#138]
### What is the problem you're trying to solve?

Developer forgetting to add `fix: { … }` to a reject case of a `testRule` which has `computeEditInfo: true`.

### Precedent

A `testRule` which has `fix: true` currently throws if the `fixed` property is missing from a reject case.

https://github.com/stylelint/jest-preset-stylelint/blob/2eec77c753b21b54efa4d58a338abcb24293a752/getTestRule.js#L113-L115

**Comment by romainmenke** [comment]
Multiple overlapping fixes are skipped and result in a null-ish `fix` field.

How do we make it easy for rule authors to test that `fix` has the expected result in all cases?

**Comment by Mouvedia** [comment]
> null-ish

Can we differentiate between missing and explicitly set to `undefined`?
i.e. `hasOwnProperty`

If not, feel free to close the issue.

**Comment by romainmenke** [comment]
I think there are these cases:
(assuming that the value matches)

| `computeEditInfo` | actual | expect | test should pass |
| --- | --- | --- | --- |
| `true` | value | value | ✅ |
| `true` | value | `undefined` | ❌ |
| `true` | `undefined ` | `undefined ` | ✅ |
| `true` | `undefined ` | value | ❌ |

-----

Now that there are some rules with support and that the feature actually works we can more easily see what is the best approach.

Before I was developing both the core change and the changes here and I wasn't yet sure how all the details would take shape.

**Comment by Mouvedia** [comment]
I meant

```js
					if (schema.computeEditInfo && !testCase.hasOwnProperty('fix')) {
						throw new Error('foo bar');
					}
```

i.e. it's not about the value of `fix` but whether it is present or not
